lib/Mojolicious.pm

```diff
@@ -175,6 +175,10 @@ sub plugin {
 
 sub server { $_[0]->plugins->emit_hook(before_server_start => @_[1, 0]) }
 
+sub worker_starting {
+  $_[0]->plugins->emit_hook(before_worker_start => $_[0]);
+}
```

Comment:
The consistent name would be `worker`. And there is no need to use more than one line.

Reply:
Hmm... adding more entries to a class which can have many heirs... we need to be careful to not choose a routine name that is likely to be conflicting with any user code already written.

As for the one-liner, agreed, it fits on 80 chars, and I can omit the trailing ';' if we make it fit on a single line,

    $ echo 'sub worker_starting { $_[0]->plugins->emit_hook(before_worker_start => $_[0]); }' | wc -c
    81 # counts the trailing hidden "\n"

Reply:
Done.